components/resources/brave_components_strings.grd

```diff
@@ -795,6 +795,7 @@
       <if expr="not is_android">
         <part file="brave_shields_strings.grdp" />
         <part file="brave_welcome_strings.grdp" />
+        <part file="new_tab_strings.grdp" />
```

Comment:
Maybe `brave_new_tab_strings.grdp` for consistency?

Reply:
Done.